shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -43,6 +43,10 @@ dropIndex
     : DROP INDEX indexName ONLINE? FORCE? ((DEFERRED|IMMEDIATE) INVALIDATION)?
     ;
 
+dropView
+    : DROP VIEW (schemaName DOT_)? viewName (CASCADE CONSTRAINTS)?
```

Comment:
`viewName` is defined in `BaseRule.g4` as the following
```
viewName
    : (owner DOT_)? name
    ;
```
so you can leave out the `(schemaName DOT_)?` here

Reply:
Oh yes 👍, Made the changes.